src/txn/txn_ckpt.c

```diff
@@ -138,18 +140,27 @@ __checkpoint_flush_tier(WT_SESSION_IMPL *session, bool force)
                 }
             }
             /* Only instantiate the handle if we need to flush. */
-            WT_ERR(__wt_session_get_dhandle(session, key, NULL, NULL, 0));
+            ret = __wt_session_get_dhandle(session, key, NULL, NULL, 0);
```

Comment:
Should we use `WT_ERR_ERROR_OK` instead with `keep` set to `true`?
```suggestion
            WT_ERR_ERROR_OK(__wt_session_get_dhandle(session, key, NULL, NULL, 0), EBUSY, true);
```

Reply:
Yes! Fixed.